lib/ts/ink_error.cc

```diff
@@ -61,19 +61,38 @@ ink_fatal(const char *message_format, ...)
   va_list ap;
 
   va_start(ap, message_format);
-  fatal_va(message_format, ap);
+  fatal_va("Fatal: ", message_format, ap);
   va_end(ap);
 
   ::exit(70); // 70 corresponds to EX_SOFTWARE in BSD's sysexits. As good a status as any.
 }
 
 void
+ink_emergency_va(const char *fmt, va_list ap)
+{
+  fatal_va("Emergency: ", fmt, ap);
+  ::exit(UNRECOVERABLE_EXIT);
+}
+
+void
+ink_emergency(const char *message_format, ...)
+{
+  va_list ap;
+
+  va_start(ap, message_format);
+  ink_emergency_va(message_format, ap);
+  va_end(ap);
```

Comment:
Hmmm. Hasn't the process already exited by this point? Maybe best to be safe, though.